purchase_requisition_auto_rfq/models/purchase_requisition.py

```diff
@@ -2,12 +2,21 @@
 
 from collections import defaultdict
 
-from odoo import _, models
+from odoo import _, api, fields, models
 
 
 class PurchaseRequisition(models.Model):
     _inherit = "purchase.requisition"
 
+    partner_rfq_ids = fields.Many2many(
```

Comment:
Following English rules, please rename this field, so its name reflects better that we're referring to partners and not to RFQs.

Also add a string:

```suggestion
    rfq_partner_ids = fields.Many2many(
        string="RFQ Partners",
```